src/main/resources/init_db.sql

```diff
@@ -0,0 +1,16 @@
+CREATE SCHEMA `internet_shop` DEFAULT CHARACTER SET utf8 ;
+
+CREATE TABLE `internet_shop`.`products` (
+  `product_id` BIGINT(11) NOT NULL,
+  `product_name` VARCHAR(225) NOT NULL,
+  `product_price` DOUBLE NULL,
+  PRIMARY KEY (`product_id`),
+  UNIQUE INDEX `products_id_UNIQUE` (`product_id` ASC) VISIBLE,
+  UNIQUE INDEX `product_name_UNIQUE` (`product_name` ASC) VISIBLE)
```

Comment:
Why product name is unique?

Reply:
It's error - I know it after morning lection